test/e2e/v1beta1/common_cbs_test.go

```diff
@@ -0,0 +1,85 @@
+// Copyright The Shipwright Contributors
+//
+// SPDX-License-Identifier: Apache-2.0
+
+package e2e_test
+
+import (
+	"context"
+
+	. "github.com/onsi/gomega"
+
+	meta "k8s.io/apimachinery/pkg/apis/meta/v1"
```

Comment:
(Super)Nit: Can we rename it `metav1`? I think this is the more widely used import name.